### PR TITLE
AC Automated Test - LRAC-8589  | master

### PR DESCRIPTION
**Pull request by catarineaguiar**
https://issues.liferay.com/browse/LRAC-8589

### CODE REVIEW

**Comment by ViniLopes20** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/EditSegments.testcase [hunk 1]
```diff
@@ -321,9 +321,49 @@ definition {
 	@ignore = "true"
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/EditSegments.testcase:321]: Hi @catarineaguiar, remove the ignore

Comment (catarineaguiar) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/EditSegments.testcase]: done

**Comment by ViniLopes20** [comment]
ci:test:sf

**Review by alexcandidos — APPROVED** [review]
Hi @catarineaguiar , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**